src/Stripe.net/Services/Accounts/AccountBankAccountOptions.cs

```diff
@@ -5,9 +5,6 @@ namespace Stripe
 
     public class AccountBankAccountOptions : INestedOptions
     {
-        [JsonProperty("object")]
-        internal string Object => "bank_account";
-
```

Comment:
This change looks like a potential bug, it's an implicit value that is required for the API to work and we don't want to force developers to set.

Reply:
Great catch, brought it back!